.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
We often want to put workspace-specific files into the `.gitignore` so they won;t be tracked. With something like prettier, whether we want the configuration in  the repo might depend on our team and if we're using it to help reinforce the team's style guide during local development.